docs/proposals/config_data.md

```diff
@@ -175,16 +184,21 @@ type VolumeSource struct {
   ConfigData *ConfigDataVolumeSource `json:"configData,omitempty"`
 }
 
+// ConfigDataVolumeSource represents a volume that holds configuration data
 type ConfigDataVolumeSource struct {
+  // A list of config data keys to project into the volume in files
```

Comment:
I see that we called this field `items` in DownwardAPIVolumeFile, but otherwise we only use that term for top-level resource lists. Could we please use a more specific field name instead? files? paths? keys?

We should add this to naming conventions:
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md#naming-conventions